matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/state/SendStateTask.kt

```diff
@@ -39,7 +39,7 @@ internal class DefaultSendStateTask @Inject constructor(
 
     override suspend fun execute(params: SendStateTask.Params) {
         return executeRequest(globalErrorReceiver) {
-            if (params.stateKey == null) {
+            if (params.stateKey.isEmpty()) {
```

Comment:
I think it is an improvement, I am not sure what would happen if `stateKey` was an empty String on the build URL (it would end by `/`.